### ISSUE TITLE
Research and explore concepts for easier legal doc viewing

### ISSUE THREAD

**Issue by noahmanger** [18F/openFEC-web-app#2123]
So that users can quickly skim through all the documents of a particular case (like an individual AO or MUR), explore concepts for how to make this easier.

---

This is an open-ended research / early design task to:
- Get a better understanding of the need
- Look at other patterns in the wild that might be useful
- Consider how this might extend to other aspects of the FEC.gov experience, such as browsing finance data, understanding reporting requirements, or other things
- If it makes sense, start sketching some concepts

**Comment by nickykrause** [comment]
Step 1: Try to line up time to talk with the target users (legal professionals) both inside and outside of the FEC. 

Update on this: I have reached out to @rhoughfec via Slack to see if she can help me out with getting people lined up for some short user interviews. (Rebecca: We can hash out the details in Slack -- I just wanted to drop a note here as well for visibility)

**Comment by rhoughfec** [comment]
@nickykrause absolutely! happy to help corral people for this.

**Comment by nickykrause** [comment]
Quick update: With @rhoughfec help, we are set to chat with a couple of the FEC policy folks tomorrow afternoon. 

We are still exploring how we can get time with legal professionals outside of the FEC.

**Comment by nickykrause** [comment]
I didn't finish this by the end of last sprint, as I had hoped I would. I'm moving it into sprint 3.1, but reducing the time estimate from 3 to 2, because some of the work has been done.

**Comment by nickykrause** [comment]
I pulled out some “themes” from the AO split pane research that we did, and I collected that information [here](https://docs.google.com/document/d/1P5lTQ8lHOJA6_AzbqSL-B8vRtzWHegiubFBnHIEgLcI/edit#).

Side note: As a part of this research, we also asked 3 external legal users about their experience with legal searching more generally, but it was very open-ended. There is a lot of interesting stuff in there that may help to inform future iterations of the search results display and filters.
